oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/userinfo/DefaultOAuth2UserService.java

```diff
@@ -157,6 +141,53 @@ private ResponseEntity<Map<String, Object>> getResponse(OAuth2UserRequest userRe
 		}
 	}
 
+	private String getUserNameAttributeName(OAuth2UserRequest userRequest) {
```

Comment:
I appreciate this cleanup. So that the feature commit has only the changes needed for the feature, will you please place these other refactors into a separate, earlier commit in the PR?

Reply:
I will do so once everything else is approved